sql/src/main/java/io/crate/executor/transport/task/elasticsearch/ESGetTask.java

```diff
@@ -122,10 +125,10 @@ public ESGetTask(UUID jobId,
             SettableFuture<Bucket> settableFuture = SettableFuture.create();
             listener = new GetResponseListener(settableFuture, extractors);
             results = ImmutableList.of(Futures.transform(settableFuture, QueryResult.TO_TASK_RESULT));
+            context = new ESJobContext("lookup by primary key", ImmutableList.of(request), ImmutableList.of(listener), results, transportAction);
```

Comment:
because the projectorChain.startProjections takes the context as argument and outside of the if/else block the projectorChain isn't available